### DISCUSSION TITLE
Welcome to robotic-transformer-pytorch Discussions!

### DISCUSSION THREAD

**Discussion by lucidrains** [lucidrains/robotic-transformer-pytorch#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lucidrains** [comment]
what do people think of RT-2 https://www.deepmind.com/blog/rt-2-new-model-translates-vision-and-language-into-action , is it worth taking time to open source?

**Reply by kyegomez** [reply]
@lucidrains Yes I have tried using your BASE PALM as the foundation for [PALM-E](https://github.com/kyegomez/PALM-E) and I have made a rough implementation here based off of your [RT-1 TokenLearner ](https://github.com/kyegomez/RT-2), [RT-2](https://github.com/kyegomez/RT-2)
